index.js

```diff
@@ -208,7 +208,7 @@ function rateLimitRequestHandler (params, pluginComponent) {
 
     let maximum
 
-    if (typeof params.max === 'number') {
+    if (typeof params.max === 'number' && !isNaN(params.max)) {
```

Comment:
Do you want me to add a test that the library fail or change the code here, I did not add a test for the `max` property for specific route as it would fail the library as it tries to call `NaN` as a function